Dockerfile

```diff
@@ -0,0 +1,82 @@
+# FROM debian:buster-slim as dqlite-lib-builder 
+FROM ubuntu as dqlite-lib-builder 
+ARG DEBIAN_FRONTEND="noninteractive"
+ENV TZ=Europe/London
+ENV LD_LIBRARY_PATH=/usr/local/lib
+ENV GOROOT=/usr/local/go
+ENV GOPATH=/go
+ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+RUN apt-get update && apt-get install -y git build-essential dh-autoreconf pkg-config libuv1-dev tcl8.6 wget
+
+WORKDIR /opt
+
+RUN git clone --depth 100 https://github.com/canonical/sqlite.git
+
+WORKDIR /opt/sqlite
+
+RUN ./configure --enable-replication && make && make install
+
+WORKDIR /opt
+
+RUN git clone https://github.com/canonical/libco.git
+
+WORKDIR /opt/libco
+
+RUN make && make install
```

Comment:
The steps to build `libco` can be dropped too, since dqlite does not depend on it anymore starting from #264.

Reply:
It fails like so if I take libco away:

```
Step 22/30 : RUN go get -d -v ./... &&     go install -tags libsqlite3 ./cmd/dqlite-demo &&     go install -tags libsqlite3 ./cmd/dqlite
 ---> Running in d73b3f842eaa
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/peterh/liner v1.2.0
go: downloading github.com/ghodss/yaml v1.0.0
go: downloading golang.org/x/sys v0.0.0-20200602225109-6fdc65e7d980
go: downloading github.com/mattn/go-sqlite3 v2.0.3+incompatible
go: downloading github.com/Rican7/retry v0.1.0
go: downloading gopkg.in/yaml.v2 v2.3.0
go: downloading github.com/spf13/cobra v1.0.0
go: downloading github.com/spf13/pflag v1.0.3
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/mattn/go-runewidth v0.0.3
# github.com/canonical/go-dqlite/internal/bindings
/usr/bin/ld: cannot find -lco
collect2: error: ld returned 1 exit status
```

Reply:
Are you sure there's not some caching involved? I'll try myself too.

Reply:
:) might be.  I'm still on it, haven't dropped it yet. Planning to get a docker-compose yaml up next so we can test cluster networking. Another pair of eyes wouldn't hurt tho. Can you try building with ‘--no-cache‘?

Reply:
I have a hunch it might have to do with line no.4 from:

https://github.com/canonical/go-dqlite/blob/master/internal/bindings/build.go

Reply:
I can reproduce the issue, looking at the cause.

Reply:
> I have a hunch it might have to do with line no.4 from:
> 
> https://github.com/canonical/go-dqlite/blob/master/internal/bindings/build.go

You're right, push a PR for that.

Reply:
Yup. Replacing:

```

package bindings

/*
#cgo linux LDFLAGS: -lsqlite3 -lraft -lco -ldqlite
*/
import "C"
```
with 

```

package bindings

/*
#cgo linux LDFLAGS: -lsqlite3 -lraft -ldqlite
*/
import "C"
```

did it. Tested locally. Any chance you can fix it there ?

Reply:
The problem should go away once https://github.com/canonical/go-dqlite/pull/120 is merged.

Reply:
That branch was merged, so your Dockerfile should work now.

Reply:
Cleaning up and will push in a bit